mergin/test/test_client.py

```diff
@@ -303,3 +303,29 @@ def toggle_geodiff(enabled):
         assert not mp3.geodiff.has_changes(mp.fpath_meta('diff'))
     else:
         assert os.path.exists(mp.fpath('base.gpkg_conflict_copy'))
+
+
+def test_list_of_push_changes(mc):
+    PUSH_CHANGES_SUMMARY = "{'base.gpkg': {'geodiff_summary': [{'table': 'gpkg_contents', 'insert': 0, 'update': 1, 'delete': 0}, {'table': 'simple', 'insert': 1, 'update': 0, 'delete': 0}]}}"
+    def toggle_geodiff(enabled):
```

Comment:
not sure if we need to toggle geodiff here but if we do, it would be better to make this helper function available for all tests